src/components/browse/singleDatasetView/singleDatasetView.tsx

```diff
@@ -35,9 +35,6 @@ const SingleDatasetView = () => {
       if (!queried) {
         setQueried(true);
         getDatasetDetails(datasetAccession, setDetails);
-        if (details === undefined) {
-          setDetails(null);
-        }
       }
```

Comment:
Good catch. That certainly didn't belong there and probably caused the error flashing I guess.